packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -183,6 +203,14 @@ class Send extends Component {
                             type='range' // In Gwei
                           />
 
+                          <TxDetails
```

Comment:
Here, we could do:
```
valid && isEstimatedTxFee(values) && !isNaN(values.amount) && <TxDetails ... />
```
so that:
- TxDetails is only mounted when necessary
- we can pass in less props to TxDetails (estimatedTxFee, token and values)

and remove this logic from the render function of TxDetails.